components/Navbar/NavbarDesktop/TabDesktop.tsx

```diff
@@ -87,9 +114,9 @@ export default function Tab({
       offset={0}
       theme={theme}
       className={clsx('cursor-default no-underline', {
-        'cursor-not-allowed': !isUnlocked,
+        'cursor-not-allowed': !isTabUnlocked,
       })}
-      disabled={!isUnlocked}
+      disabled={!isTabUnlocked}
```

Comment:
It seems like the "disabled" behavior for the top page selection tabs is inconsistent. I found it to not work correctly on the first few pages, CH1INT1-CH1GEN1

Reply:
this seems consistent in all chapters for the first 3 pages in each chapter